src/ch/imedias/rsccfx/view/HeaderView.java

```diff
@@ -54,7 +57,8 @@ private void layoutForm() {
 
   private void initFieldData() {
     //populate fields which require initial data
-
+    // FIXME: move those to "layoutForm" and only keep the ones that actually populate fields with initial data
```

Comment:
@Gyssim move those to "layoutForm" and only keep the ones that actually populate fields with initial data